AzureMS/src/constants/GameConstants.java

```diff
@@ -7960,5 +7956,50 @@ public static boolean isIllium(final int job) {
     public static boolean isMichael(final int job) {
         return job == 5000 || (job >= 5000 && job <= 5112);
     }
-    
+
+    //methods to initialise hashmap for MapleCharacter.checkForceShield()
+    //EXPERIMENTAL; lazy intialisation
+    // contents: <jobID, equipID>
+    public static final Map<Short, Integer> checkSecondary = new HashMap<Short, Integer>(){{
```

Comment:
CHECK_SECONDARY

Reply:
I was just following the conventions in the constants for the server but sure 🤙